### ISSUE TITLE
What should be the range for Mass 1 position slider?

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#79]
@arouinfar is it ok to have a zero kilometer mark in this simualtion? I think in gravity-firce-lab the actual range is a little less than 10 meters 0.2 meters to 9.8 meters.

Not specifically for this issue, but in other circumstances @emily-phet mentioned to me that PhET sims generally avoid zero values. Since this is a kilometer mark (a position value) and not the actual distance value used in the equation, perhaps it is ok that the kilometer mark is zero?

**Here's a screenshot with both sims at extremes:**
<img width="747" alt="screen shot 2018-11-14 at 11 47 42 pm" src="https://user-images.githubusercontent.com/1902116/48518570-5b418c80-e86a-11e8-8529-085a8a04f507.png">

**Putting masses to their minimums thankfully does not increase position range in either sim:**
<img width="746" alt="screen shot 2018-11-14 at 11 54 43 pm" src="https://user-images.githubusercontent.com/1902116/48518168-03eeec80-e869-11e8-90f2-bc504dd94cc1.png">

**Questions:**
1. Is it necessary for the sims to have similar ranges, though at different scales (meters and kilometers)?
If yes:
2. What is the optimal minimum position value for Mass 1 position slider in gavity-frorce-lab-basics?
3. What is the optimal minimum position value for m1 position slider gravity-force-lab?

@arouinfar, please re-assign to me after you share your thoughts. Thanks!

**Comment by arouinfar** [comment]
@terracoda due to https://github.com/phetsims/gravity-force-lab-basics/issues/75, the maximum distance between the masses was reduced to 9.6 km in GFLB and 9.6 m in GFL. The max distance remains 10 cm/pm in Coulomb's Law because the the charges do not change radius, so the figures don't end up getting pushed off-screen.

For all of these sims, the underlying gridlines are equally spaced, in the sense that two gridlines is the distance between tick marks on the ruler (this is the distance moved with the arrow keys). 

So, to answer your questions...
> @arouinfar is it ok to have a zero kilometer mark in this simulation?

I don't see any particular problem with starting at zero in the ISLC sims. In fact, I think it'll make more sense. Otherwise, a user might try to move a mass located at 0.2 km/m further to the left, which wouldn't be possible. Having the position value start at zero feels more natural.

> Is it necessary for the sims to have similar ranges, though at different scales (meters and kilometers)?

I think it's good to provide the largest possible range of distances, which map to the underlying grid. The maximum separation will be 96 gridlines (GFL/GLFB) or 100 gridlines (CL). I think it's perfectly acceptable for GFL/GFLB and CL to have slightly different ranges, as all three sims have maximized the distance between the masses/charges while keeping the leaning figures still fully visible on-screen.

**Comment by terracoda** [comment]
@arouinfar, great. I agree, let's start positions at zero, that does indeed seem like the most natural way to do it.

So the position ranges in GFL will be  0 meters to 9.6 meters, in GFL:B 0 km to 9.6 km (no change needed), and 0 cm to 10 cm in CL when we get there.

And the distance between the objects, which is the meaningful value for these equations, is never a zero value.

Thanks @arouinfar!

**Comment by terracoda** [comment]
@mbarlow12, no need to change range of mass 1 sphere's position as it is correct at 0 km to 9.6 km. I will close this issue and open a new issue for https://github.com/phetsims/gravity-force-lab/issues repo.
